src/cli/commands/files/add.js

```diff
@@ -135,6 +135,10 @@ module.exports = {
       default: false,
       describe: 'Only chunk and hash, do not write'
     },
+    chunker: {
+      default: 'size-262144',
+      describe: 'Chunking algorithm to use, formatted like [default, size-{size}, rabin, rabin-{avg}, rabin-{min}-{avg}-{max}]'
```

Comment:
This is really a minor thing: I've seen that the Go implementation can parse `default` as value, but also the CLI help of `ipfs add --help` doesn't show this option. I'd remove `default` from this list and also the code below. Less options, less bugs :)